net-libs/libpsl/libpsl-0.19.1.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="C library for the Public Suffix List"
+DESCRIPTION="Libpsl can be used by browsers and other web clients to avoid privay-leaking 'supercookies' or 'super domain' certificates, highlight parts of the domain in a user interface, and sort domain lists by site.'"
```

Comment:
Please adjust this according to the [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips)